portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,29 +20,31 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  ArrayList<Comment> jsonArray = new ArrayList<Comment>();
```

Comment:
super, super, super nit: rename this to commentsList? This technically isn't in JSON format during its usage.